config/locales/fr.yml

```diff
@@ -0,0 +1,8 @@
+fr:
+  activemodel: &errors
+    errors:
+      messages:
+        invalid_email_address: 'se semble pas être une adresse email valide'
```

Comment:
'se semble pas être une adresse email valide' must be 'ne semble pas être une adresse email valide'